src/pages/en/reference/adapter-reference.md

```diff
@@ -4,7 +4,7 @@ title: Astro Adapter API
 i18nReady: true
 ---
 
-Astro is designed to make it easy to deploy to any cloud provider for SSR (server-side rendering). This ability is provided by __adapters__, which are [integrations](/en/reference/integrations-reference/).
+Astro is designed to make it easy to deploy to any cloud provider for [SSR (server-side rendering)](/en/guides/server-side-rendering/). This ability is provided by __adapters__, which are [integrations](/en/reference/integrations-reference/). See the [SSR guide](/en/guides/server-side-rendering/#adding-an-adapter) to learn how to use an existing adapter.
```

Comment:
```suggestion
Astro is designed to make it easy to deploy to any cloud provider for SSR (server-side rendering). This ability is provided by __adapters__, which are [integrations](/en/reference/integrations-reference/). See the [SSR guide](/en/guides/server-side-rendering/#adding-an-adapter) to learn how to use an existing adapter.
```
This seems a little much, linking to the same page twice in the same sentence? I'm going to take editorial liberty and remove one, so that I can merge this otherwise perfectly fine PR quickly.   :)

Could go either way, but I'm thinking that since this is the adapter reference page, I'll keep the one you added about learning how to use an existing adapter.